teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -28,6 +28,7 @@ abstract class TeamCityInstance {
     abstract fun users(): UserLocator
     abstract fun buildAgents(): BuildAgentLocator
     abstract fun buildAgentPools(): BuildAgentPoolLocator
+    abstract fun testOccurrences(): TestOccurrencesLocator
```

Comment:
I'm not sure about `testOccurrences` name, it looks unclear for me. I don't see "test occurrence" words in TeamCity Web UI and [its documentation](https://confluence.jetbrains.com/display/TCD18/TeamCity+Documentation). Unfortunately we already have `TestOccurrence` interface but it's possible to rename it as well.

What do you think about renaming this function (and the `Locator` interface) to something different, e.g. `testResults()`?